static/logo.svg

```diff
@@ -1 +1,10 @@
S0FeZCjaCN1FHCEAAAhUIMGWMxMX66BBJSwUh6eQSj8fj/07TJE+nZ3vigomKaApTwTpxCooJAQhAoCcC3pQxebAXIwQwMGUDJC09qVHesspoyuPx+BfTvqrq+vsDIiWu5G1NzgYBCEAAAhCYCUiQcc79SuJSNcCbShA02g5JyzgSJxo2j6bIU+kZTal8A0biB8nKOP5GTSEAAQg0J8B6F5IWjcnH1TKRtDSXlKIFeDwe/2eaJtamVE5QfH9k3UpRE+fkEIAABCBwREA6e865r35w4j0JTW82QNJy5Ol9/d9LUn7vzRatlXdOVthSvC8XorQQgAAE7BJ4PB4fZFGltYBLfcZIwEha+tYmf12Kc+5P/La9386L7D/0bVmUHgIQgAAEzBKYn+/CYv2G0zDosJ3vsJG09CVJy0iKLJ4nSTlv74U14n2RfV8WRWkhAAEIQGBIArLIcpqmz4UDI4vTSYyy2QBJi16pmhfOL2tSmO6l2+95OKReV6JkEIAABCCwRWBerM9OY7o7Gdk6/j0nqSQtW15c/7joxvzkedndi4c6dqAf845gj/rWwhUhAAEIQAACGQlID4RtktVN4SBZ8TqDsiYio8lzqkQC4SgKU7360gmSlURD52sQgAAEINAXAZKXvjokPY+cnC17X57UZ2n9BGVei8IzUrzE+azNtvw+yUqfPkipIQABCEDgJAGSF5KXlh2u2LVPmjBfPyAgPj4/wPHfJCh2/J1k5cDw+TcEIAABCNgkIL0apo3Z6dDEkoFejtn0sPK1WkZPHo/Hv6ZpkjUoLJTvdPRkz1dJVsr7EleAAAQgAIEOCJC8kLjsdZhq/K8DN2laRNli2Bs5ITkxmJjE/IxkpanbcXEIQAACENBKgOSF5CXWcapxTKtP1CiXN1ryvq3wNE3LlC5GTQZJTkIfI1mp4XlcAwIQgAAEuifgJS88pHLQTlPYiSr9uXunCSrgJyLegxglGfmPTOGa/1gIj3/5uwiK3n4W/Q3MiY8QgAAEIAABCOwRkI7XNE2fpmkieaFz5Xeusr/fs8Ot/0nnbp42JVOnSvzJOhFJNF7+vMRjSUB4ngk+ctUv3p9gL3q7ZeschwAEIAABCEAgkcDj8fjgnPte+o4752eKGjaADYxgA6KnoquJEszXIAABCEAAAhA4Q2BOXr6M0KmgjnSesQFsILcNOOe+kKyciTp8FwIQgAAEIHCDwOPx+Kdz7rfcAZ3z0UnEBrABizYgeim6eUN2+SkEIAABCEAAAlcJyKICnvVCJ9NiJ5M6YdcZbOCNncCuRhd+BwEIQAACEChAYF60/5F1L3T0MnT0ri5o5ncshldhA7MOfmRxfYFgwykhAAEIQAACuQgwdYzEhcQFGxjRBpgCliuKcB4IQAACEIBARQIydYwtk+m8jth5pc5D2f2yZTHPV6kYX7gUBCAAAQhAoAgBdh0bqhOnYooOiQM2V9IG2AWsSKjgpBCAAAQgAAEdBB6Px8/zwn0eWMn6A5IbbKA3G2BhvY5QQikgAAEIQAACdQjIAtV59OVrybuhnJu77dgANnDXBpxzX0WvWFhfJz5wFQhAAAIQgIBKAvPal8/sPEbn8m7nkt9jQ7lsYNajz6JPKoWTQkEAAhCAAAQg0I7A4/H4heljdDxzdTw5D7Z00gaW6V88BLJdGODKEIAABCAAgX4IeNPHvpzsdPQ2R57ysq4DG2hsA8uieqZ/9RMjKCkEIAABCEBAHYFl62Smj3HXnAQWG8hlA7OefGL6lzrJp0AQgAAEIACB/gnMD678dZomdh9rfHc6V+eR85CI1LIBSVRk+qnoSP9qSA0gAAEIQAACEOiCwLL+hREYOr21Or1cpz9bm/VBFtT/3IWwUUgIQAACEIAABOwSkA7JNE3sQMboC+tDsIG/SFTsaj01gwAEIAABCJghQALT391wRjBos7s2QKJiRsKpCAQgAAEIQGA8AiQwdIbvdob5vV4bIlEZT9OpMQQgAAEIQMA8gTmB+SRPt6YjqrcjStvQNns2MPuv7PrFGhXzqk0FIQABCEAAAoMT8J4D8xs7kdFJ3usk87/m9rE88PEDz1EZXLipPgQgAAEIQGB0ArIF6ryQn1EYFnKzmL+xDcyjKbLjF1sTjy7O1B8CEIAABCAAgTiB+WGWH51zMgpDBxYG2EAFG5j97SMPe4zrEkchAAEIQAACEIDALgFGYUjcSF7z24C3NoXRlF0F4p8QgAAEIAABCEDgJIF5LcwvTCXL34klMbDN1Jvy9QtrU04KD1+HAAQgAAEIQAACdwiQxNjuaJNIXW9fkpQ7ysJvIQABCEAAAhCAQEECSxLjnPt17rSxHqLCegiSi+vJRS52JCkFhYVTQwACEIAABCAAgZIE5iRGdiaT58N8YXvl9p3rXJ30wc8j2xCLPcvzUv7JdK+SKsK5IQABCEAAAhCAQAMC8lC8x+PxgdEYEpheEh9vFEWelcJDHRvoBpeEAAQgAAEIQAACzQnMO5TJaAwPu2Q6WesphTKKInb4PorS3DkoAAQgAAEIQAACEICATgLyrAo/kXHOfe/lrjzl7GcESezKT1DE6HR6BKWCAAQgAAEIQAACEOiGwJzIfJy3XJY1Ba3vynP9TtpgXoPyeZomeYAjz0bpxuspKAQgAAEIQAACEDBAYF4jI8+OeV/sz6jM2IncPHqyLJKXZ6KwBsWAn1MFCEAAAhCAAAQgYJJAkMzIFsyMzHQyKpIygibtKZs5zGtPSE5MejGVggAEIAABCEAAAoMS8LZhlmlmy+jM15SOMt+pO4oju3Z5WwvLzl1sLzyo31JtCEAAAhCAAAQgAIGZwDw6Ix3jD5LQeEmN3NWXkZo3Epcsicv7c07m0ZL3qVzziMmSmDClC6+EAAQgAAEIQAACEIBADgJekiOJzvufLPReEp7wdemkJ7w22S1tWQ+SUL5nohHWcVnovvCYX0lCchgc54AABCDQiMD/B+aGL+yADhfgAAAAAElFTkSuQmCC"/>
```

Comment:
Looks like this is an embedded png

Maybe we should use [this](https://raw.githubusercontent.com/denolib/high-res-deno-logo/master/deno_hr_circle.svg) instead?

Reply:
Fixed